app/buck2_event_publisher_proto/event_publisher.proto

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under both the MIT license found in the
+ * LICENSE-MIT file in the root directory of this source tree and the Apache
+ * License, Version 2.0 found in the LICENSE-APACHE file in the root directory
+ * of this source tree.
+ */
+
+syntax = "proto3";
+
+import "data.proto";
+
+package event_publisher;
+
+message BuckEventRequest {
+  // A trace-unique 64-bit identifying the stream.
+  uint64 stream_id = 1;
+
+  buck.data.BuckEvent event = 2;
+};
+
+message BuckEventResponse {
+  // A trace-unique 64-bit identifying the stream.
+  uint64 stream_id = 1;
```

Comment:
> Instead of having stream_id we can just create a new grpc call for each stream, that would be more natural.

- @stepancheg

Reply:
Actually, to partially respond to Stiopa on this: That seems hard to practically implement on the buck2 side, since we may create a number of different instances of the client for each command. And even if we didn't do that, it seems unwise to prevent ourselves from doing that in the future